src/app/services/command/command.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+import { CommandRequest } from '../../models/angular-command-request';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
please check if we need this line.
I don't use it in my projects. 
I don't understand what it's for.

Reply:
[https://angular.io/guide/dependency-injection#create-an-injectable-heroservice](url)

>   // we declare that this service should be created
>   // by the root application injector.
>

Reply:
Is `'root'` the default value?
[https://angular.io/api/core/Injectable](url)